src/WaypointManager.lua

```diff
@@ -59,11 +69,22 @@ function WaypointManager:RemovePlayerWaypoint()
     self.LMP:RemoveMapPing(MAP_PIN_TYPE_PLAYER_WAYPOINT)
 end
 
-function WaypointManager:SetMeasurementWaypoint(x, y)
+function WaypointManager:SetMeasurementWaypoint(localX, localY)
     -- this waypoint stays invisible for others
     self.suppressCount = self.suppressCount + 1
     self.LMP:SuppressPing(MAP_PIN_TYPE_PLAYER_WAYPOINT)
-    self:SetPlayerWaypoint(x, y)
+
+    local x, y = 1, 1
+    if(math.abs(pwx - x) < 10000) then x = pwx + 10000 end
```

Comment:
this throws an error "operator - is not supported for nil - number". Looks like pwx and pwy should be changed to localX and localY?

Reply:
Yes and no. This does not work in zones where the position of the player is "stuck". A fix point far away 10000,10000 works.